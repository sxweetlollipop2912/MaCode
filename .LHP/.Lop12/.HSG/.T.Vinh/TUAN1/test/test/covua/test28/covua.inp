30
xd10011d010d0d0dd1.0101101d0d1
dx101d0000.1d101d1d01000111100
00x0.dd0001.1d1d01110dd1d10d11
111xdd0d100d0dd11.d10000001101
10.dx1110d0d0ddd0dd1011101100d
0ddd0xd11d00d10d0010d001d01101
01d10dx10.10.dddd11000101110d1
d11d000x00d01d1101011ddd001111
11101011xd0..0d110d.d0d1110d01
0111dd.1dx.d10d0001100000dd001
1.01110d1.x0111ddd10110dd1d1d1
d0.dd111.d1x00110d110dd001111d
1d011d.0.001x000d01ddd0ddd10d1
d0ddd0dd11011xd01d01ddd0010010
110dd1d0dd001dx0d1.d0d101001.1
d0d0ddd001d0111x1d0011d0.d0011
dd1011d101d1d0d0x1d0.111.ddd10
000.d10011dd1d0d0xd1d10d100d10
.d0dd001d00001.1ddx.01d0dd0000
11000110.010d0d110.x.0000d1d1d
00111d10d101dd10.d1.x0111d10dd
11d1011d110dddd000011x100d1000
01d1010dd11d1d0d01d100xdd00110
0101001d01d1d1110d1101dxd0d0.0
10d11d0101d1d10..0d101ddx1ddd0
000101010d00d01dd1dddd110x1101
d01000001dd00111d110001dd0x1d0
10d01010d1001101dd1d1101d00x0.
d10111d011d0d0.00010d10.d1d1xd
0100d000000d0100111dd111101.dx