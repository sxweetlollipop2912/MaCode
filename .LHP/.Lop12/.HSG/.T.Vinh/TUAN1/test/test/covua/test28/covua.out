9 11 12 17 21
